DirectProgramming/DPC++FPGA/ReferenceDesigns/gzip/Zlib_License.txt

```diff
@@ -1,25 +0,0 @@
-zlib License
```

Comment:
@JoeOster, why was this file deleted?

This isn't the boilerplate license file checked in with every code sample -- it's a 3rd party library dependence.